server/src/main/java/tabletop/domain/exceptions/BadRequestException.java

```diff
@@ -0,0 +1,15 @@
+package tabletop.domain.exceptions;
+
```

Comment:
if you add this line here:
@ResponseStatus(value = HttpStatus.BAD_REQUEST)
you will be getting a 400 error (bad request) instead of a 500.
However, the way of handling these events should be consistent, e.g. the same for events.
I use the ResponseUtils.badRequest method but it's made to pass a list of more than one error.
If there's one error message only, like for example when a record doesn't exist, an exception could be good.